### PR TITLE
update the image override

### PR DESCRIPTION
**Pull request by hchenxa**
refer https://issues.redhat.com/browse/ACM-5985

### CODE REVIEW

File: operator/config/samples/operator_v1alpha3_multiclusterglobalhub.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: MulticlusterGlobalHub
 metadata:
   # annotations:
     # mgh-pause: "true"
-    # mgh-image-repository: quay.io/<quay_io_repo>
```
Comment (clyang82) [inline operator/config/samples/operator_v1alpha3_multiclusterglobalhub.yaml:6]: why do you remove `quay_io_repo`?

Comment (hchenxa) [reply operator/config/samples/operator_v1alpha3_multiclusterglobalhub.yaml:6]: this should used to replace the registry server, I did not think we should support to replace the registry org.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request closed without merging**